src/components/Annotation/Annotation.scss

```diff
@@ -0,0 +1,128 @@
+@import '../../common.scss';
```

Comment:
Are any of these styles duplicates of existing annotations CSS? I would expect most of this to be here already if we're going for a similar look/DOM layout to current annotations.

Reply:
I'm merging the existing annotations CSS with any CSS needed to combine the subcomponents.  The plan is to delete the duplicate css from Annotator.scss when i actually use this component in the UI